src/jrd/version.h

```diff
@@ -0,0 +1,151 @@
+ *
+ *  All Rights Reserved.
+ *  Contributor(s): ______________________________________.
+ */
+
+#ifndef JRD_VERSION_H
+#define JRD_VERSION_H
+
+#define STRINGIZE_AUX(x)	#x
+#define STRINGIZE(x)		STRINGIZE_AUX(x)
+
+
+#ifdef RC_TARGET_engine13
+#define VER_FILEDESC "Engine plugin"
+
+#elif defined RC_TARGET_fb_lock_print
+#define VER_FILEDESC "Lock Print tool"
+
+#elif defined RC_TARGET_fbguard
+#define VER_FILEDESC "Guardian"
+
+#elif defined RC_TARGET_fbrmclib
+#define VER_FILEDESC "RM/COBOL client interface"
+
+#elif defined RC_TARGET_firebird
+#define VER_FILEDESC "Server main executable"
+
+#elif defined RC_TARGET_fbsvcmgr
+#define VER_FILEDESC "Service Manager"
+
+#elif defined RC_TARGET_fbtrace
+#define VER_FILEDESC "Trace plugin"
+
+#elif defined RC_TARGET_fbtracemgr
+#define VER_FILEDESC "Trace Manager"
+
+#elif defined RC_TARGET_gbak
+#define VER_FILEDESC "Gbak tool"
+
+#elif defined RC_TARGET_gfix
+#define VER_FILEDESC "Gfix tool"
+
+#elif defined RC_TARGET_gpre
+#define VER_FILEDESC "Gpre tool"
+
+#elif defined RC_TARGET_gsec
+#define VER_FILEDESC "Gsec tool"
+
+#elif defined RC_TARGET_gsplit
+#define VER_FILEDESC "Gsplit tool"
+
+#elif defined RC_TARGET_gstat
+#define VER_FILEDESC "Gstat tool"
+
+#elif defined RC_TARGET_ib_util
+#define VER_FILEDESC "UDF support library"
+
+#elif defined RC_TARGET_instclient
+#define VER_FILEDESC "Install Client tool"
+
+#elif defined RC_TARGET_instreg
+#define VER_FILEDESC "Install Registry tool"
+
+#elif defined RC_TARGET_instsvc
+#define VER_FILEDESC "Install Service tool"
+
+#elif defined RC_TARGET_fbintl
+#define VER_FILEDESC "International Characters support library"
+
+#elif defined RC_TARGET_isql
+#define VER_FILEDESC "Interactive Query tool"
+
+#elif defined RC_TARGET_legacy_auth
+#define VER_FILEDESC "Legacy Auth plugin"
+
+#elif defined RC_TARGET_legacy_usermanager
+#define VER_FILEDESC "Legacy User Manager plugin"
+
+#elif defined RC_TARGET_nbackup
+#define VER_FILEDESC "Physical Backup Manager"
+
+#elif defined RC_TARGET_qli
+#define VER_FILEDESC "QLI tool"
+
+#elif defined RC_TARGET_srp
+#define VER_FILEDESC "SRP User Manager plugin"
+
+#elif defined RC_TARGET_udf_compat
+#define VER_FILEDESC "UDF compatibility library"
+
+#elif defined RC_TARGET_udr_engine
+#define VER_FILEDESC "User Defined Routines engine"
+
+#elif defined RC_TARGET_fbclient
+#define VER_FILEDESC "Client library"
+
+#elif defined RC_TARGET_build_msg
+#define VER_FILEDESC "Build Message file tool"
+
+#elif defined RC_TARGET_codes
+#define VER_FILEDESC "Generate Error Codes tool"
+
+#elif defined RC_TARGET_gpre_boot
+#define VER_FILEDESC "Gpre boot tool"
+
+#elif defined RC_TARGET_udrcpp_example
+#define VER_FILEDESC "UDR C++ example"
+
+#elif defined RC_TARGET_fbudf
+#define VER_FILEDESC "UDF library"
+
+#elif defined RC_TARGET_ib_udf
+#define VER_FILEDESC "UDF library"
+
+#else
+#define VER_FILEDESC "SQL Server"
+
+#endif
+
+
+#ifdef NDEBUG
+#define VER_DBG "release"
+#else
+#define VER_DBG "debug"
+#endif
+
```

Comment:
I seriously don't think we need that. Debug builds can be seen on developer machines only and the file size/performance usually explains it ;-) 

But if you insist, then please show it for debug versions only. No need to see "release" in 99.999% of all builds.

Reply:
Ok, I'll remove "release" and leave "debug"